LocationManagerExample/LocationManagerExample/INTUViewController.m

```diff
@@ -216,13 +210,11 @@ - (IBAction)desiredAccuracyControlChanged:(UISegmentedControl *)sender
 - (IBAction)timeoutSliderChanged:(UISlider *)sender
 {
     self.timeout = round(sender.value);
+    NSString *timeString = [NSString stringWithFormat:@"Timeout: %ld seconds", (long)self.timeout];
     if (self.timeout == 0) {
-        self.timeoutLabel.text = [NSString stringWithFormat:@"Timeout: 0 seconds (no limit)"];
-    } else if (self.timeout == 1) {
-        self.timeoutLabel.text = [NSString stringWithFormat:@"Timeout: 1 second"];
-    } else {
-        self.timeoutLabel.text = [NSString stringWithFormat:@"Timeout: %ld seconds", (long)self.timeout];
+        timeString = [NSString stringWithFormat:@"%@ (no limit)", timeString];
     }
+    self.timeoutLabel.text = timeString;
```

Comment:
I see the intent of trying to condense the code here, but this change eliminates the handling of 1 second (singular). I don't think this is a significant enough improvement to warrant making any change here.